rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py

```diff
@@ -33,11 +33,23 @@ def getImageUploadAndColor():
         print(request.get_header(head))
         if headers.get(head) not in request.get_header(head):
             abort(406, 'Incorrect headers')
+    
+    colors = ['green', 'blue', 'yellow', 'red', 'purple', 'pink', 'black', 'brown', 'dark red', 'dark yellow', 'dark blue',
+    'light red', 'light green', 'grey', 'orange', 'white']
     colorRequest = request.forms.get('color')
     imageRequest = request.files.get('image')
-    if not (colorRequest or imageRequest):
+    userHeightRequest = request.forms.get('userHeight')
+    wallHeightRequest = request.forms.get('wallHeight')
+    
+    if not (colorRequest and imageRequest):
         abort(400, 'Bad request')
 
+    if not colorRequest in colors:
+        abort(407, 'Bad color input')
+        
+    if not (userHeightRequest.isnumeric() and wallHeightRequest.isnumeric()):
```

Comment:
This should be an or statement, if just one of them is wrong it should throw an error. Should not depend on both of them being wrong to throw an error

Reply:
Whoops my bad, we talked about this